pages/jedi/index.js

```diff
@@ -29,12 +29,12 @@ function JEDIAbout () {
 				}}
 			/>
       <div className={styles.container}>
-        <div className="text-center">
+        <div className={`text-center ${styles['mt-2']}`}>
           <Image src={tempJediBanner} alt="a temporary banner image with a diverse group of people experiencing technology" />
         </div>
         <div className="text-center">
           <h1 className={styles['hero-title']}>ACM | JEDI</h1>
-          <p>justice, equity, diversity, inclusion</p>
+          <p className={styles['hero-subtitle']}>justice, equity, diversity, inclusion</p>
```

Comment:
What does `hero` mean

Reply:
Common SEO / marketing term for above-the-fold images & headers! Here's a [spammy-ish article about it](https://www.optimizely.com/optimization-glossary/hero-image/).